packages/switch/README.md

```diff
@@ -0,0 +1,55 @@
+# React Switch
+
+A React version of an [MDC Switch](https://github.com/material-components/material-components-web/tree/master/packages/mdc-switch).
+
+## Installation
+
+```
+npm install @material/react-switch
+```
+
+## Usage
+
+### Styles
+
+with Sass:
+```js
+import '@material/react-switch/index.scss';
+```
+
+with CSS:
+```js
+import "@material/react-switch/dist/switch.css";
+```
+
+### Javascript Instantiation
```

Comment:
how does one get a hold of the value? if its checked/not checked

Reply:
Added a note to the README, basically same way as in text field